tests/test_types.py

```diff
@@ -46,18 +46,34 @@ def test_item_type():
     assert 'invalid choice: x' in str(e.value)
 
 
-@mock.patch('planet.scripts.types.get_asset_types',
-            new=mock.Mock(return_value=DEFAULT_ASSET_TYPES))
-def test_asset_type():
-    check = convert_asserter(AssetType())
-    check('visual*', ['visual', 'visual_xml'])
-    check('*data_*', ['metadata_aux', 'metadata_txt'])
-    check('analytic', ['analytic'])
-    check('analytic,visual', ['analytic', 'visual'])
```

Comment:
Each of these checks should be its own test function. When serialized an early failure can hide a later from being seen until the tests are run again.